src/apps/vpn/ui/screens/settings/ViewDNSSettings.qml

```diff
@@ -29,18 +29,37 @@ VPNViewBase {
     }
 
     _viewContentData: ColumnLayout {
-        spacing: VPNTheme.theme.windowMargin * 2
-        Layout.fillWidth: true
+        spacing: VPNTheme.theme.windowMargin * 1.5
         Layout.leftMargin: VPNTheme.theme.windowMargin
         Layout.rightMargin: VPNTheme.theme.windowMargin
-        Layout.topMargin: VPNTheme.theme.windowMargin / 2
+
+        Loader {
+            objectName: "DNSSettingsViewInformationCardLoader"
+            active: Qt.platform.os === "ios" && VPNController.state !== VPNController.StateOff
```

Comment:
The best would be to expose the silent switch without disconnect property from the controller.

Basically: VPNController.disconnectNeededForSwitching (or a better name).

The impl should be done in controller.{cpp,h} and from there to the ControllerImpl via an override const method.